app-misc/kryoflux-dtc/files/kryoflux-ui

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# Change the directory, to find the `dtc` symlink and launch the Kryoflux UI
+cd /usr/share/kryoflux-ui && java -jar kryoflux-ui.jar
```

Comment:
Looks like a case for `make_wrapper` in eutils.

Reply:
I didn't know about that. Thanks, changed now.